src/main/java/io/reactivex/observers/TestObserver.java

```diff
@@ -678,6 +751,65 @@ public final boolean awaitTerminalEvent(long duration, TimeUnit unit) {
     }
 
     /**
+     * Sets the initial fusion mode if the upstream supports fusion.
+     * @param mode the mode to establish, see the {@link QueueDisposable} constants
+     * @return this
+     */
+    public final TestObserver<T> setInitialFusionMode(int mode) {
```

Comment:
The problem with that is that `Observable.test()` would need to return it or one would need to cast it to the internal implementation. We can move the constants into the public space. The other option would be to move test consumers into the test source set that would remove the `test` operators as well. Damn lack of extension methods.

Also a less ugly version would be to turn `test()` methods into calls of `to(Function)` and provide TestHelper.newTest(int fusionMode)` that return some custom `TestObserver`.